### ISSUE TITLE
Don't strip initial or trailing space in `script` tags so that `no-multiple-empty-lines`, etc. will work as desired

### ISSUE THREAD

**Issue by brettz9** [vuejs/eslint-plugin-vue#1325]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** 7.10.0
- **eslint-plugin-vue version:** 7.0.1
- **Node version:** 12.16.2
- **Operating System:** Mac OSX 10.15.7

**Please show your full configuration:**

```json5
{
  'no-multiple-empty-lines': ['error', { max: 2, maxBOF: 0, maxEOF: 0 }]
}
```

**What did you do?**	

```vue
<script>

import { computed } from 'vue'

</script>
```

**What did you expect to happen?**

To get linting errors for extra whitespace at the beginning and end of the script tag.

**What actually happened?**

No errors were reported

**Repository to reproduce this issue**

N/A

**Comment by ota-meshi** [comment]
Thank you for this issue!

Are you saying that you want to get the following code with auto-fix of `no-multiple-empty-lines` rule?

```vue
<script>
import { computed } from 'vue'
</script>
```

If so, the `maxBOF` and `maxEOF` options of the `no-multiple-empty-lines` rule affect the begin and end of file. It does not affect `<script>` tags.

I think we need to add new rule to this plugin to check the style you want.
e.g. `vue/block-tag-newline` rule that enforce line breaks style after opening and before closing block-level tags.

**Comment by brettz9** [comment]
I see. Yes, a new rule would be great, thanks!
